api/v1alpha6/openstackcluster_webhook.go

```diff
@@ -115,6 +115,10 @@ func (r *OpenStackCluster) ValidateUpdate(oldRaw runtime.Object) error {
 		r.Spec.APIServerLoadBalancer.AllowedCIDRs = []string{}
 	}
 
+	// Allow changes to the availability zones.
+	old.Spec.ControlPlaneAvailabilityZones = []string{}
+	r.Spec.ControlPlaneAvailabilityZones = []string{}
+
```

Comment:
Whoops! That's quite an oversight.

Please could you add unit tests for these to `openstackcluster_webhook_test.go`?

Reply:
roger!